cmd/juju/machine/upgradeseries.go

```diff
@@ -160,6 +161,13 @@ func (c *upgradeSeriesCommand) Run(ctx *cmd.Context) error {
 			return errors.Trace(err)
 		}
 	}
+	if c.prepCommand == CompleteCommand {
```

Comment:
Is this clearer as a switch?

switch c.prepCommand {
  case PrepareCommand:
   ...
  case CompleteCommand:
  ...
}

Reply:
I generally switch if there are more than two cases.